src/components/EditModal/index.tsx

```diff
@@ -14,6 +14,8 @@ import { auth, db } from '../..'
 const DEFAULT_MAP_STYLE = 'mapbox://styles/mapbox/outdoors-v11'
 const WAIT_BEFORE_CLOSE = 2000
 
+type ModalState = 'default' | 'saving' | 'saved'
```

Comment:
`default` might not be the best choice of word, but I couldn't really think of another...open to suggestions or we can leave it as is